### ISSUE TITLE
Create webhook when deploying via SSH and given a token

### ISSUE THREAD

**Issue by oskapt** [rancher/rio#972]
If a service is started with an SSH key, but a token was provided for creating a webhook, we should be able to infer the repo name and create the webhook, right? This doesn't appear to be happening, yet no error is reported (#971).

**Comment by cbron** [comment]
Assuming you mean that this fails with valid creds:`rio run -p 80/http -n rio-test --build-clone-secret=gitcredential-ssh --build-webhook-secret=githubtoken  git@github.com:user/repo`
because repo url is not in standard format. I confirmed that. If you check the gitwatcher it does show an error but we don't bubble that up to the service. We already have a ticket for that so this ticket can stay for handling the ssh style url `git@github.com:user/repo` ->`github.com/user/repo`

**Comment by oskapt** [comment]
Yes, correct as you described it. If you already have an issue for this that will be resolved soon, you can close this. If it won't be resolved soon, please consider this a request to notify the user that the requested action failed.
